src/components/atoms/jump-to-menu/jump-to-menu.scss

```diff
@@ -0,0 +1,47 @@
+@use "../../../sass/settings";
+@use "../../../sass/typography";
+
+.jump-menu-container {
+  border-left: 2px solid settings.$color-less-light-grey;
+  margin: auto;
+  max-width: 69.625rem;
+}
+
+.jump-menu-list {
+  list-style: none;
+  margin-left: -2px;
+  margin-top: 21.5px;
+  padding-left: 0;
+}
+
+.jump-menu-list__item {
+  border-left: 2px solid settings.$color-less-light-grey;
+  margin-bottom: 24px;
+  min-width: 150px;
+  padding-left: 20px;
+
+  &:hover {
+    border-left: 2px solid settings.$color-primary;
+  }
+}
+
+.jump-menu-list__item:last-child {
+  margin-bottom: 0;
+}
+
+.jump-menu-list__item--active {
+  border-left: 2px solid settings.$color-primary;
+  margin-bottom: 24px;
+  min-width: 150px;
+  padding-left: 20px;
+}
+
+.jump-menu-list__link {
+  @include typography.navigation-heading();
+
+  text-decoration: none;
+}
+
+.jump-menu-list__item--active .jump-menu-list__link {
```

Comment:
Can we reduce the specificity on this one?

Reply:
Do you want to move the `--active` flag down to the link? We wouldn't be able to just remove the parent class as the logic currently is.

Reply:
Ok. Well leave it as it is and I will look at it when I review all of the CSS